pkg/compiler/gen.go

```diff
@@ -134,6 +134,8 @@ func (comp *compiler) genSyscall(n *ast.Call, argSizes []uint64) *prog.Syscall {
 		Ret:         ret,
 		Attrs:       attrs,
 	}
+	checkCompressedArgs(result)
```

Comment:
This check needs to happen in compiler.check method. Generation is not supposed to produce any errors. At this stage we can only panic, which is not good handling of wrong descriptions. + we can't print file:line and we can't test this.